test/saved_outputs.py

```diff
@@ -366,6 +368,8 @@
 Total mass (amu):      339672.2974
 Number of atoms:       56057
 Number of residues:    18015
+Residue set:           ['ALA', 'ARG', 'ASN', 'ASP', 'CYS', 'GLN', 'GLU', 'GLY', 'HSD', 'ILE', 'LEU', 'LYS', 'MET', 'PHE', 'PRO', 'SER', 'THR', 'TIP3', 'TRP', 'TYR', 'VAL']
+Residue count:          'ALA': 13, 'ARG': 9, 'ASN': 5, 'ASP': 14, 'CYS': 2, 'GLN': 4, 'GLU': 12, 'GLY': 10, 'HSD': 5, 'ILE': 12, 'LEU': 11, 'LYS': 6, 'MET': 5, 'PHE': 6, 'PRO': 10, 'SER': 9, 'THR': 6, 'TRP': 5, 'TYR': 4, 'VAL': 11,'TIP3': 17856
 System volume (ang^3): 615109.34
```

Comment:
yes. We can count the length of string and and split it (if > 80?).

I will submit another PR (soon).